ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -6993,6 +6993,40 @@ def test_torch_instance_atan2_(
     )
 
 
+# bitwise_not_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="bitwise_not_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
+        num_arrays=2,
```

Comment:
Hey @jakubbober, is there a need for generating two arrays here?

Reply:
Hi, I actually don't think so, but I wanted to match the logic from `test_torch_instance_bitwise_not_` (it also generates two arrays).

Reply:
@bipinKrishnan should I change it or keep it consistent with the other function?

Reply:
Okay, makes sense :slightly_smiling_face: